### PR TITLE
Feature/upload image

### PR DESCRIPTION
**Pull request by hanhEvizi**

### CODE REVIEW

File: .env.qa [hunk 1]
```diff
@@ -0,0 +1 @@
+X_API_KEY = live_ZQOlINBtwABczolmhjUt4V0K9GlZPR1pPSkJRC3Dxo8dIy2ziv9vbmfgAGCPT3Px
```
Comment (kahnext) [inline .env.qa]: Critical problem: Do NOT push cred key onto Github

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+node_modules
+package-lock.json
+coverage
+.env.qa
```
Comment (trung-tran-everest) [inline .gitignore]: missing a new line

File: __tests__/images/post_images.spec.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+const api = request.agent(BASE_URL);
+
+// Get file path of image file
+const filePath = path.join(path.dirname(path.dirname(__dirname)), 'files/dog.jpg');
+
+let imageIds: string[] = [];
+
+describe("'POST' - /images/upload - Post image", () => {
+  it("TDA_06 Verify that the user can upload an image with a valid file path", async () => {
+    const response = await api
```
Comment (kahnext) [inline __tests__/images/post_images.spec.ts]: this step is used in other place, why don't we make it as a uploadImage method for reuse?

File: .env.qa (hunk 1 above)
Comment (trung-tran-everest) [inline .env.qa]: should not push your api key to repo

File: __tests__/images/post_images.spec.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    expect(response.statusCode).toBe(201);
+    expect(response.body.sub_id).toEqual(imageSubId);
+    expect(image.original_filename).toEqual(FILENAME);
+  }, TIMEOUT);
+
+  // Delete test images
+  afterEach(async () => {
+    for (const imageId of imageIds) {
+      await api.delete(`/v1/images/${imageId}`).set({ "x-api-key": X_API_KEY });
```
Comment (kahnext) [inline __tests__/images/post_images.spec.ts]: 1. Why don't we make a deleteImage method for reuse?
2. Correct order of request step: set cred key first, then call `delete` API

File: __tests__/images/post_images.spec.ts (hunk 3 above)
Comment (trung-tran-everest) [inline __tests__/images/post_images.spec.ts]: you should write a helper method for upload file action

File: __tests__/images/post_images.spec.ts (hunk 4 above)
Comment (trung-tran-everest) [inline __tests__/images/post_images.spec.ts]: you should write a helper method for delete file action

File: __tests__/images/post_images.spec.ts [hunk 5]
```diff
@@ -0,0 +1,69 @@
+import request from "supertest";
+import path from "path";
+import { BASE_URL, X_API_KEY, TIMEOUT, FILENAME } from "../../helper/constants";
+import { randomString } from "../../helper/common.helper";
+
+const api = request.agent(BASE_URL);
+
+// Get file path of image file
+const filePath = path.join(path.dirname(path.dirname(__dirname)), 'files/dog.jpg');
```
Comment (trung-tran-everest) [inline __tests__/images/post_images.spec.ts]: you have a constant variable for dog.jpg file, why don't use it?

**Review by evizi-baolonglehoang — APPROVED** [review]

**Pull request merged**